android/build.gradle

```
@@ -11,35 +11,63 @@ android {
             assets.srcDirs = ['assets']
             jniLibs.srcDirs = ['libs']
         }
+        //improved by cinnamon17
 
-        instrumentTest.setRoot('tests')
+    }
+    packagingOptions {
+        exclude 'META-INF/robovm/ios/robovm.xml'
+    }
+    defaultConfig {
+        applicationId "com.jumpersuper"
```

En su momento le tuve que cambiar el nombre a la aplicación por `es.danirod.jddprototype.android` para poder subirla a la Play Store (aunque me la tumbaron hace un año por "poco útil"), de modo que si el applicationId ahora hay que ponerlo en el build.gradle, el valor de `applicationId` debería ser cambiado a `"es.danirod.jddprototype.android"`.

```suggestion
        applicationId "es.danirod.jddprototype.android"
```